android/src/main/kotlin/com/frontegg/flutter/Utils.kt

```diff
@@ -14,37 +14,64 @@ interface ActivityProvider {
 
 val Context.constants: FronteggConstants
     get() {
-        val packageName = this.packageName
-        val className = "$packageName.BuildConfig"
-        try {
-            val buildConfigClass = Class.forName(className)
-
-            // Get the field from BuildConfig class
-            val baseUrl = safeGetValueFromBuildConfig(buildConfigClass, "FRONTEGG_DOMAIN", "")
-            val clientId = safeGetValueFromBuildConfig(buildConfigClass, "FRONTEGG_CLIENT_ID", "")
-
-            val applicationId =
-                safeGetNullableValueFromBuildConfig(buildConfigClass, "FRONTEGG_APPLICATION_ID", "")
-
-            val useAssetsLinks =
-                safeGetValueFromBuildConfig(buildConfigClass, "FRONTEGG_USE_ASSETS_LINKS", true)
-            val useChromeCustomTabs = safeGetValueFromBuildConfig(
-                buildConfigClass, "FRONTEGG_USE_CHROME_CUSTOM_TABS", true
-            )
-
-            return FronteggConstants(
-                baseUrl = baseUrl,
-                clientId = clientId,
-                applicationId = applicationId,
-                useAssetsLinks = useAssetsLinks,
-                useChromeCustomTabs = useChromeCustomTabs,
-                bundleId = this.packageName,
-            )
-        } catch (e: ClassNotFoundException) {
-            Log.e(TAG, "Class not found: $className")
-            throw e
+        val mainActivity = getLaunchActivityName(this)
+        Log.d(TAG, "packageName: ${packageName}, mainActivity: $mainActivity")
```

Comment:
Remove this comment

Reply:
Sorry
Do you mean Log.d(TAG, "packageName: ${packageName}, mainActivity: $mainActivity")
Or Log.e(TAG, "Class not found: $className")?

Reply:
Nevermind